app/assets/javascripts/src/updateOtherEvidencePage.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+
+    'use strict';
+
+    var root = this, $ = root.jQuery;
+
+    if (typeof VOA === 'undefined') {
+        root.VOA = {};
+    }
+
+    var UpdateOtherEvidencePage = function (){
+
+        $("#continue-button").click(function() {
+          $('#continue').click();
+        });
+
+        var options = "input[value='lease'],input[value='license'],input[value='serviceCharge'],input[value='stampDutyLandTaxForm'],input[value='waterRateDemand'],input[value='otherUtilityBill'],input[value='landRegistryTitle']";
```

Comment:
also here, listing inputs by their value attribute is brittle. We can list all options within a specific `fieldset`

something like this
```javascript
$('fieldset#fieldset-id input[type="checkbox"]')
```